ReadMe.md

```diff
@@ -67,13 +67,13 @@ One liner: `./flash_core1_main.sh`
    ```
 3. Prepare the container:
    ```sh
-   docker compose up -d
+   docker-compose up -d
```

Comment:
Hi ) 

https://docs.docker.com/engine/reference/commandline/compose/

Old versions does not have it?

Reply:
Hey!

Nice, docker now includes compose as a sub-command! But it works only since 3.4.0 as [Compose V2](https://docs.docker.com/compose/cli-command/). `docker-compose` seems more compatible.

Reply:
I used `docker compose` intentionally to popularize new Docker versions. 

Will reconsider this.

Reply:
Ok, nice point. If docker has this feature now, I think my proposal no longer needed.